libipl/p10/ipl0.C

```diff
@@ -374,28 +387,137 @@ static int ipl_alignment_check(void)
 	return -1;
 }
 
+static int initialize_and_check_clock_chip(uint8_t& clock_count)
+{
+	struct pdbg_target *clock_target;
+	enum pdbg_target_status status;
+	uint8_t data;
+	int i2c_rc = 0;
+	int rc = 0;
+
+	// initialize output variable.
+	clock_count = 0;
+
+	ipl_log(IPL_DEBUG, "Istep: soft reset clock, and verify clock status register\n");
+	pdbg_for_each_class_target("oscrefclk", clock_target) {
+		if(!ipl_is_functional(clock_target)){
```

Comment:
This need functional state update for refclcok targets in updatehw model.  considering code consistency  think better use the general design direction setting target as functional in updatgw mdele and continue this direction.

Reply:
Done